README.md

```diff
@@ -217,7 +167,7 @@ redisClusterReset(clusterContext);
 
 ## Cluster asynchronous API
 
-Hiredis-vip comes with an cluster asynchronous API that works easily with any event library.
+Hiredis-cluster comes with an cluster asynchronous API that works easily with any event library.
 Now we just support and test for libevent and redis ae, if you need for other event libraries,
 please contact with us, and we will support it quickly.
```

Comment:
Is this correct? Isn't it "libevent" and "libev"?

Reply:
correct, but updated